project1/client/input_parser/input_parser.h

```diff
@@ -0,0 +1,105 @@
+#ifndef PROJECT1_INPUT_PARSER_H
+#define PROJECT1_INPUT_PARSER_H
+
+#include "../../wire_protocol/wire_protocol.h"
+#include "../../server/file_handler/my_file_handler.h"
+#include "ftp_messages.pb.h"
+#include <sstream>
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace client::input_parser {
+
+    /**
+     * @brief Assists in mapping input commands for FTP client to corresponding
+     * protobuf requests.
+     *
+     * @note There's definitely some redundancy here. Will likely change
+     * this class to have templated CreateReq functions in project 2.
+     */
+    class InputParser {
+    public:
+        enum ReqType {GETF, PUTF, DEL, LS, CD, MKDIR, PWD, QUIT};
```

Comment:
Use `enum class` here instead. Strongly-typed enums in C++ [are generally preferable to C enums](https://stackoverflow.com/a/12581154/1516662).